src/components/AChat/AChatForm.vue

```diff
@@ -154,6 +156,12 @@ export default {
     onKeyCommand: function (event) {
       if (event.ctrlKey && event.shiftKey && event.code === 'Digit1') {
         this.openElement()
+      } else if (event.code === 'ArrowUp' || event.code === 'ArrowDown') {
+        this.selectCommand(event.code)
+        event.preventDefault()
```

Comment:
Better to move `event.preventDefault()` at the top of the function body.

Reply:
if i'll move preventDefault() at the top of function body it will totally break input's behavior.
So i'm gonna leave it unchanged.